client/src/components/MapWithMarker.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+
+const MapWithAMarker = withGoogleMap(props => (
+	<GoogleMap defaultZoom={8} defaultCenter={{ lat: 32.699635, lng: 35.303547 }}>
```

Comment:
How did you decide this defaultCenter lat and long? I think this should be declared as a `const defaultCenter` at the top of the file with a comment stating where this location is.